config/webpack.prod.config.js

```diff
@@ -16,6 +16,9 @@ const path = require('path');
 const PostCssAutoprefixerPlugin = require('autoprefixer');
 const PostCssRTLCSS = require('postcss-rtlcss');
 
+// Reduce CSS file size by ~70%
+const purgecss = require('@fullhuman/postcss-purgecss');
```

Comment:
The codebase hasn't been updated in years, apparently.  Are there no more active alternatives?

Reply:
Hum... This installs `@fullhuman/postcss-purgecss` version > 5.x which was publi[shed 6 months ago in npm](https://www.npmjs.com/package/@fullhuman/postcss-purgecss) hence the last git commit https://github.com/FullHuman/purgecss/commits/main/packages/postcss-purgecss repo is quite acitve. 
Probably you were looking to something else, as it's easy to find pkgs with similar name